### DISCUSSION TITLE
Secondary Button Theming

### DISCUSSION THREAD

**Discussion by jcbjoe** [twilio-labs/paste#2898] [Ideas] [answered]
Hi All!

Firstly apologies if this is the wrong place for such a request.

I have been creating a custom theme using "CustomizationProvider" and I have noticed there are no tokens to change the secondary background colour. Im wanting to change the background colour of secondary buttons. I see there is "colorBackgroundPrimary" but no "colorBackgroundSecondary". Is this intentional or something that was missed out?

Thanks,
Joe

**Comment by nkrantz** [comment, marked as answer]
Hi @jcbjoe - this is the right place! Great to hear that you're using the Customization Provider. 

We name our tokens semantically, so colorBackgroundPrimary is what we typically use in product for a background color, but you can use any of the background color tokens to customize your secondary buttons and it should work just fine. 

Please let us know if you have any more questions around the Customization Provider or Paste! 

Just a note: the design systems team will be out of office next week during the company break, and will be slower to respond to GitHub Discussions and other questions through the new year. We'll be back to our regular response times on January 2, 2023

**Reply by nkrantz** [reply]
Here's a CodeSandbox for more guidance https://codesandbox.io/s/brave-williamson-futb1i
